puppet/services/network/contrail-tsn.yaml

```diff
@@ -21,15 +33,15 @@ parameters:
   NeutronMetadataProxySharedSecret:
     description: Metadata Secret
     type: string
-  VrouterPhysicalInterface:
+  ContrailVrouterPhysicalInterface:
```

Comment:
the base for this pull-request for ocata was taken from the Michael's  review to openstack master, where such prepending done for most variables.